bundles/org.openhab.automation.jythonscripting/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon:addon id="pidcontroller" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<addon:addon id="jythonscripting" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Nice finding! 👍 I have picked this change into a separate PR, #16527, so that it can be backported to 4.1.